### DISCUSSION TITLE
Question on setup

### DISCUSSION THREAD

**Discussion by balajeek** [gtxaspec/wz_mini_hacks#97] [General]
I did follow the instructions and installed on wyze cam v3 floodlight last week.  
I did get voice message that its updating the firmware and camera booted and I was able to see on wyze app.
I am unable to ssh as shown in the document and from wyze app->advanced settings there is no item called RTSP, is this suppose to be there or no matter what the RTSP should work by using the rtsp url mentioned ? (I tried and it wouldn't work either)

So i am trying to understand to troubleshoot further.  Thanks.

btw: do i need to leave the sd card in there all time for this hack to work or can i take it out after the first initial boot after update?
I will eventually connect this to frigate dvr to capture recordings.

**Comment by virmaior** [comment]
1. RTSP from wz_mini_hacks will not appear in the wyze app.
2. if you set the parameters for RTSP in wz_mini_hacks, then the url 
rtsp://login:password@IP_ADDRESS:8554/unicast
should work
e.g. rtsp://admin:password@192.168.9.103:8554/unicast 
in an application like VLC
3. must always leave the SD card in as this does not **install** anything on your camera itself. It loads itself because the bootloader scans for a file on the SD card during the boot process before it starts using its own software.

**Reply by balajeek** [reply]
So once i booted with the hack files in the sd card for first time, and then i remove the sd card and make changes to the config by adding the rtsp username password and enabling few other settings, i just use the same sd card to boot?

**Reply by gtxaspec** [reply]
right, you always use the sd card to boot the camera, and you always keep it in while the camera is in use.

**Reply by balajeek** [reply]
thanks, will try it out when i get home.

**Reply by balajeek** [reply]
home now, and i put the sdcard and rebooted.  
Now i can SSH and wondering how can i change the config file to enable rtsp , not sure where the location is.
thanks.

**Reply by gtxaspec** [reply]
/opt/wz_mini/wz_mini.conf

**Reply by balajeek** [reply]
I modified the conf and enabled rtsp hi res.
I am able ssh (slow though)
I am able to connect the camera from wyze app
but unable to do rtsp
  - from vlc i do rtsp://admin@192.168.9.40:8554/unicast, it then prompts for password, I enter them, then error comes unable to open rtsp, check for error logs.
  - from vlc if i try wrong password purposely, it does keep prompting to enter correct password, so username/password i have is correct but not sure whats wrong.

**Reply by gtxaspec** [reply]
what happens if you do `rtsp://admin:password@192.168.9.40:8554/unicast`, entering the password in the url?

**Reply by balajeek** [reply]
its the same error

**Reply by gtxaspec** [reply]
@balajeek can you run `gather_wz_logs.sh` and upload the file here? I'll take a look whats going on

**Comment by balajeek** [comment]
thanks for your help, here is the log file that generated by gather_wz_log.sh
[log_gather_2022-06-19_T1749.tar.gz](https://github.com/gtxaspec/wz_mini_hacks/files/8936093/log_gather_2022-06-19_T1749.tar.gz)

**Reply by gtxaspec** [reply]
download the latest version and try again, there are some fixes in the new version.

**Reply by balajeek** [reply]
okay will give it a try and let you know today.

**Reply by balajeek** [reply]
I just did download and redid everything, from wyze app work fine, but from vlc i still get the same error.
How can i check the downloaded is the latest so i can make sure?
thanks.

**Reply by gtxaspec** [reply]
use https://github.com/gtxaspec/wz_mini_hacks/archive/refs/heads/master.zip

**Reply by balajeek** [reply]
trying it right now ...

**Reply by balajeek** [reply]
Well, i tried and still did not work.  got the same error "unable to open the MRL rtsp://..."

**Reply by virmaior** [reply]
clear out 
```
+ RTSP_HI_RES_FPS=15
+ RTSP_HI_RES_MAX_BITRATE=1024
+ RTSP_HI_RES_TARGET_BITRATE=1024
+ RTSP_HI_RES_ENC_PARAMETER=2
```
and see if that helps. 

My understanding is that those changes are always going to be a bit buggy because the stream is controlled by iCamera and any switch is a re-encode

**Reply by balajeek** [reply]
I tried what you mentioned, still got the same error.

i have another v3 camera (not the floodlight) should i give try on that ?
just wondering what could be the problem!
